src/scss/3_components/navigation/_main-nav.scss

```diff
@@ -7,7 +7,13 @@
 //
 // A list of links to the site's top-level pages.
 //
+// The max-width of the main-navigation section is set using the $max-grid-width variable, declared in the
+// 'init/_variables.scss' partial.
+//
+// The background and border color of elements make use of color variables, set in the 'init/_colors.scss' partial.
+//
 // :hover - A menu item that is being hovered over.
+// .active-trail - A menu item that represents the current active page
```

Comment:
remove this modifier class for now - need to have a look to see if we need to show a multi level nav example, might be able to get a away with a single level list